components/searchable_user_list/searchable_user_list.tsx

```diff
@@ -169,8 +170,8 @@ class SearchableUserList extends React.PureComponent {
     render() {
         let nextButton;
         let previousButton;
-        let usersToDisplay;
-        const {formatMessage} = this.props.intl;
+        let usersToDisplay = this.props.users;
```

Comment:
You might be able to change this to `let usersToDisplay: UserProfile[];`. Right now, it looks a bit confusing since we set `usersToDisplay` multiple times in a row

Reply:
When I change this to UserProfile[], it says that "usersToDisplay" is used before being assigned.